feat: ingress support

ffMathy opened the pull request:
Fixes #68.

Rbillon59 commented:
Thank you for your contribution ! 

I will test on my instance your changes, in the meantime, could you double check the `DOCS.md` file and update accordingly the documentation as you removed most of the parameters (basic auth, certs, protocol etc..) ?

ffMathy commented:
Ah yes, I'll get those removed from the docs soon 🙏

ffMathy commented:
I corrected the documentation now! ❤️

ffMathy commented:
Added a few more things - hope that's OK:
- Fixed a reference to the wrong icon file. Icon was broken.
- Added Dependabot for automatic PRs to update the Dockerfile when new versions come out of n8n.
- Renamed `docker-entrypoint.sh` to `n8n-entrypoint.sh`, because that's what it was actually doing.

ffMathy commented:
Just cleaned up a minor thing as well. Got rid of nginx template and environment variable substitution, but now I honestly can't find anything else to clean up.

Rbillon59 commented:
Seems all OK, works like a charm.

Very well done, thank you !

Dependabot is not needed tho, renovate does the job already :)

Rbillon59 commented:
Could you remove the dependabot configuration so I can approve and merge your contribution ?

ffMathy commented:
Yeah I must admit I added the Dependabot before I knew that Renovate was already configured.

Left it there though because Renovate is external, and Dependabot is inbuilt, so I thought that'd be better.

Will remove again 👍

ffMathy commented:
There we go! :heart:

ffMathy commented:
You are welcome! I added another PR here: #161

I believe it could also be a nice addition.
